app/views/posts/index.html.erb

```diff
@@ -1 +1,28 @@
-<p>Here is the list of posts for a given user</p>
\ No newline at end of file
+<div>
+<div class='user-card'>
+ <%= image_tag @user.photo, :class => 'user-showed-image', alt: 'User image' %>
+ <div class='user-showed-details'>
+ <h3><%= @user.name %></h3>
+      <p>Number of posts: <%= @user.posts_counter.nil? ? 0 : @user.posts_counter %></p>
```

Comment:
- [ ] Even ı have some posts, the number of post is 0. Kindly fix it.

Reply:
Hello @ahmetbozaci, thank you for the feedback.
About this issue, calling the method `update_post_counter` updates the number displayed on the view.
because I just checked again it's working as expected.
with:
`Post.find(post_id).update_post_counter`